### DISCUSSION TITLE
Multi config doesn't work for me

### DISCUSSION THREAD

**Discussion by Askancy** [tailwindlabs/tailwindcss#10980] [Help] [answered]
I have been using tailwind in the public template for several years now. I wanted to start using it in the admin as well, so I started multi config.

To start, I cloned my **tailwind.config.js** and **renamed** it to **tailwind.admin.config.js**.

I use Laravel, so the Resources/css I have:

app.css:

```
@config "../../tailwind.config.js";
@tailwind base;
@tailwind components;
@tailwind utilities;
```

admin.css

```
@config "../../tailwind.admin.config.js";
@tailwind base;
@tailwind components;
@tailwind utilities;
```

tailwind.admin.config.js

```
/** @type {import('tailwindcss').Config} */
module.exports = {
    content: [
        "./resources/views/admin/**/*.blade.php",
        "./resources/**/*.js",
        "./resources/**/*.vue",
        "./node_modules/flowbite/**/*.js",
    ],

```

tailwind.config.js

```
/** @type {import('tailwindcss').Config} */
module.exports = {
    content: [
        "./resources/views/front/**/*.blade.php",
        "./resources/**/*.js",
        "./resources/**/*.vue",
        "./node_modules/flowbite/**/*.js",
    ],
```

The webpack.mix.js I modified it like this:

```
mix.js('resources/js/app.js', 'public/js')
    .postCss('resources/css/app.css', 'public/css', [
        require("tailwindcss")
    ]);

    
mix.js('resources/js/admin.js', 'public/js')
    .postCss('resources/css/admin.css', 'public/css', [
        require("tailwindcss")
    ]);
```

now however when i write some css inside admin.css, via npm run watch i find no errors however it does not generate the admin.css file in public directory.

![image](https://user-images.githubusercontent.com/789702/231008801-1826ec4b-0bf2-483f-a053-02ea8d588c96.png)

Where am I going wrong?

**Comment by wongjn** [comment, marked as answer]
It seems like there is no `/js/admin.js` compiled either – perhaps the second `.js()` call chain has no effect? You could try chaining it all together:

```js
mix
  .js('resources/js/app.js', 'public/js')
  .postCss('resources/css/app.css', 'public/css', [
    require("tailwindcss")
  ])
  .js('resources/js/admin.js', 'public/js')
  .postCss('resources/css/admin.css', 'public/css', [
    require("tailwindcss")
  ]);
```
